### PR TITLE
Feat/parent request order

### PR DESCRIPTION
**Pull request by claresudbery**
#### Tests
Does this PR include tests for the changes introduced?
- [ ] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: server/models/approvals.js [hunk 1]
```diff
@@ -99,6 +102,9 @@ module.exports = (sequelize, DataTypes) => {
           as: 'User',
           attributes: ['FullNameValue']
         }
+      ],
+      order: [
+        ['createdAt', 'DESC']
```
Comment (chadtomkiss-madetech) [inline server/models/approvals.js:107]: This query will only return 1 result so doesn't need ordering.

File: server/models/approvals.js [hunk 2]
```diff
@@ -120,6 +126,9 @@ module.exports = (sequelize, DataTypes) => {
...
```
Comment (chadtomkiss-madetech) [inline server/models/approvals.js:131]: Same here.. shouldn't need to do any ordering for this query.

File: src/app/features/dashboard/home-tab/home-tab.component.html [hunk 3]
```diff
@@ -79,7 +79,7 @@ <h3 class="govuk-heading-s govuk-!-margin-top-1">Related content</h3>
         <li *ngIf="canViewDataPermissionsLink">
           <a (click)="setDataPermissions($event)" href="#">Set data permissions</a>
         </li>
-        <li *ngIf="canLinkToParent">
+        <li *ngIf="canLinkToParent && !workplace.isParent">
```
Comment (aaron-russell) [inline src/app/features/dashboard/home-tab/home-tab.component.html:82]: This is done on the back end already

File: src/app/features/dashboard/home-tab/home-tab.component.html [hunk 4]
```diff
@@ -90,7 +90,7 @@ <h3 class="govuk-heading-s govuk-!-margin-top-1">Related content</h3>
         <li *ngIf="canRemoveParentAssociation">
           <a (click)="removeLinkToParent($event)" href="#">Remove link to my parent organisation</a>
         </li>
-        <li *ngIf="canBecomeAParent && !linkToParentRequestedStatus">
+        <li *ngIf="canBecomeAParent && !linkToParentRequestedStatus && !workplace.isParent">
```
Comment (aaron-russell) [inline src/app/features/dashboard/home-tab/home-tab.component.html:93]: Same here

File: src/app/features/dashboard/home-tab/home-tab.component.html (hunk 3 above)
Comment (chadtomkiss-madetech) [reply src/app/features/dashboard/home-tab/home-tab.component.html:82]: @aaron-russell The permission check always returns true when you're an admin. So this change is to hide the links from Admins if the workplace they are viewing is already a parent.

Comment (aaron-russell) [reply src/app/features/dashboard/home-tab/home-tab.component.html:82]: @chadtomkiss-madetech that plays into what I said the other day, Permissions is broken... That shouldn't be possible

**Pull request merged**